### PR TITLE
Ismaelcrespo add hometown

### PR DESCRIPTION
**Pull request by ismaelcrespof**
This is a pull request :palm_tree:  :mouse:

### CODE REVIEW

File: ismaelcrespo-madrid.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Madrid, Spain
+
+### Places to eat
+- Churrería
+- Restaurante Pizzería
+
+### Tourism
+- Parque Warner
+- Plaza de Cibiles
```
Comment (ismaelcrespof) [inline ismaelcrespo-madrid.md]: I do not agree

**Comment by ismaelcrespof** [comment]
Good

**Comment by ismaelcrespof** [comment]
A request should be made

File: ismaelcrespo-madrid.md (hunk 1 above)
Comment (ismaelcrespof) [inline ismaelcrespo-madrid.md]: Line note

**Comment by jchue** [comment]
I agree with the second person.

**Comment by ismaelcrespof** [comment]
I do not

File: ismaelcrespo-madrid.md (hunk 1 above)
Comment (jchue) [inline ismaelcrespo-madrid.md]: I agree with you not agreeing with yourself.

File: ismaelcrespo-madrid.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Madrid, Spain
+
+### Places to eat
```
Comment (ismaelcrespof) [inline ismaelcrespo-madrid.md]: Commenting line 2

**Pull request merged**